packages/fxa-event-broker/lib/db/firestore.ts

```diff
@@ -8,8 +8,13 @@ import { Datastore } from './index';
 
 class FirestoreDatastore implements Datastore {
   private db: Firestore;
+  private prefix: string;
 
   constructor(config: Settings, firestore?: Firestore) {
+    this.prefix = config.prefix || 'fxa-eb-';
```

Comment:
there's an already default in the config, do you need the fallback `fxa-eb`?

Reply:
Because in TypeScript the key here is not guaranteed to exist. I'll change the Settings type to avoid it.